File: back_end/petmily_pjt_temp/src/main/java/com/pjt/petmily/domain/user/dto/UserLoginInfoDto.java
package com.pjt.petmily.domain.user.dto;

import com.pjt.petmily.domain.shop.entity.Item;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UserLoginInfoDto {
    String userEmail;
    String userToken;
    String userNickname;
    String userProfileImg;
    String userLikePet;
    Item userBadge;
    Item userRing;
    Item userBackground;
    Long userLoginDate;
    Boolean userIsSocial;
    LocalDateTime userAttendance;
}

File: mobile/Petmily/app/src/main/java/com/petmily/repository/dto/UserLoginInfoDto.kt
package com.petmily.repository.dto

import java.sql.Date

data class UserLoginInfoDto(
    var userId: Long = 0L,
    var userEmail: String = "",
    var userPw: String = "",
    var userToken: String = "",
    var userNickname: String = "",
    var userRegion: String = "",
    var userProfileImg: String? = "",
    var userLikePet: String = "",
    var userPoint: Int = 0,
    var userLoginDate: Date = Date(System.currentTimeMillis()),
    var userIsSocial: Boolean = false,
    var role: String = "",
)
